hub/schema/bbox_array.py

```diff
@@ -0,0 +1,67 @@
+"""
+License:
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0.
+If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+"""
+
+from typing import Tuple
+
+from hub.schema.features import Tensor
+
+
+class BBoxArray(Tensor):
```

Comment:
Why do we need an additional BBoxArray? Change the name to BBox and maybe add a category/subtype.